Update nullable-operators.md (#25348)
dotnet/docs: docs/fsharp/language-reference/symbol-and-operator-reference/nullable-operators.md

---
title: Nullable Operators in Queries
description: Learn about the nullable operators that are available in the F# programming language, for use in query expressions.
ms.date: 05/16/2016
---
# Nullable Operators in Queries

Nullable operators are binary arithmetic or comparison operators that work with nullable arithmetic types on one or both sides. Nullable types arise frequently when you work with data from sources such as databases that allow nulls in place of actual values. Nullable operators are used frequently in query expressions. In addition to nullable operators for arithmetic and comparison, conversion operators can be used to convert between nullable types. There are also nullable versions of certain query operators.

> [!NOTE]

> > Nullable operators are only used in [query expressions](../query-expressions.md). If you do not use query expressions, you do not need to

Reviewer: ```suggestion
> Nullable operators are only used in [query expressions](../query-expressions.md). If you don't use query expressions, you don't need to
```
